calico/v3.28.0/apiserver/rockcraft.yaml

```diff
@@ -0,0 +1,52 @@
+name: calico-apiserver
+summary: Calico apiserver ROCK image.
+description: >
+  This rock is a drop in replacement for the
+  docker.io/calico/apiserver:v3.28.0 image.
+version: v3.28.0
+license: Apache-2.0
+
+base: ubuntu@24.04
+build-base: ubuntu@24.04
+platforms:
+  amd64:
```

Comment:
Why don't we build this ROCK also for ARM?

Reply:
I believe we are building the `-moonray` flavor for arm64. I think building this for ARM might be better. For ARM runners we'd need to use/move to a branch from this repo

Reply:
> we'd need to use/move to a branch from this repo

I can't do that either. However, I got access to a Cloudbase ARM machine so I built the images there.

Reply:
Why can't you do that? You should be part of the Canonical Kubernetes team and hence have the permissions.

Reply:
My request to join the Github Canonical organization hasn't been approved yet.
![image](https://github.com/user-attachments/assets/cbc21b38-efd6-4671-a5ea-679c23c3c9c3)